package.json

```diff
@@ -21,7 +21,8 @@
   ],
   "scripts": {
     "build": "babel js/ux-capture.js | uglifyjs >js/ux-capture.min.js",
-    "build:clean": "rm js/ux-capture.min.js"
+    "build:clean": "rm js/ux-capture.min.js",
+    "test": "mocha"
```

Comment:
Do we care about using mocha vs. jest here? cc @mmcgahan

Reply:
Let's stick with Jest if possible just to keep the toolset consistent with the rest of the platform

Reply:
Shouldn't be a problem from what I understand.